### PR TITLE
Updated READMEs for kubernetes example pipelines

### PR DESCRIPTION
**Pull request by mkbhanda**
Signed-off-by: mkbhanda <malini.bhandaru@intel.com>

## Description

Updated common READMEs and example specific READMEs for the 4 example pipelines for Kubernetes

## Issues

List the issue or RFC link this PR is working on. If there is no such link, please mark it as `n/a`.

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)

## Dependencies

List the newly introduced 3rd party dependency if exists.

## Tests

Describe the tests that you ran to verify your changes.

### CODE REVIEW

File: DocSum/kubernetes/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+Please refer to [Docker Xeon README](https://github.com/opea-project/GenAIExamples/blob/main/DocSum/docker/xeon/README.md) or
+[Docker Gaudi README](https://github.com/opea-project/GenAIExamples/blob/main/DocSum/docker/gaudi/README.md) to build the OPEA images. 
+These will be available on Docker Hub soon, simplifying installation.
+
+
+## Deploy the RAG pipeline
+This involves deploying the application pipeline custom resource. You can use docsum_xeon.yaml if you have just a Xeon cluster or docsum_gaudi.yaml if you have a Gaudi cluster.
+
+1. Setup Environment variables
+
```
Comment (ashahba) [inline DocSum/kubernetes/README.md:25]: I'd add a line here and warn users that:
- **Note:** The following environment variables are specific to user's environment setup and need to be obtained by them prior to export.

File: DocSum/kubernetes/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+export http_proxy=${your_http_proxy}
+export https_proxy=${your_http_proxy}
+export LLM_MODEL_ID="Intel/neural-chat-7b-v3-3"
+export HUGGINGFACEHUB_API_TOKEN=${your_hf_api_token}
+export ns=${docsum}
+```
+
+2. Create namespace for the application and deploy it
+```bash
+kubectl create ns $(ns)
```
Comment (ashahba) [inline DocSum/kubernetes/README.md:37]: This has to be:
```
kubectl create ns ${ns}
```

File: DocSum/kubernetes/README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+kubectl apply -f $(pwd)/docsum_xeon.yaml
+```
+
+3. GMC will reconcile the custom resource and get all related components/services ready. Confirm the service status using below command
+```bash
+kubectl get service -n $(ns)
```
Comment (ashahba) [inline DocSum/kubernetes/README.md:43]: This one too:
```
kubectl get service -n ${ns}
```

File: DocSum/kubernetes/README.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+```
+
+4. Obtain the custom resource/pipeline access URL
+
+```bash
+kubectl get gmconnectors.gmc.opea.io -n $(ns)
```
Comment (ashahba) [inline DocSum/kubernetes/README.md:49]: `$(ns)` -> `${ns}`

File: DocSum/kubernetes/README.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+```bash
+kubectl create deployment client-test -n chatqa --image=python:3.8.13 -- sleep infinity
+```
+
+6. Access the pipeline using the above URL from the client pod and execute a request
+
+```bash
+export CLIENT_POD=$(kubectl get pod -l app=client-test -o jsonpath={.items..metadata.name})
+export accessUrl=$(kubectl get gmc -n $ns -o jsonpath="{.items[?(@.metadata.name=='docsum')].status.accessUrl}")
+kubectl exec "$CLIENT_POD" -n $ns -- curl $accessUrl  -X POST  -d '{"query":"Text Embeddings Inference (TEI) is a toolkit for deploying and serving open source text embeddings and sequence classification models. TEI enables high-performance extraction for the most popular models, including FlagEmbedding, Ember, GTE and E5."}'  -H 'Content-Type: application/json'
```
Comment (ashahba) [inline DocSum/kubernetes/README.md:65]: We only need one space before `-X`, `-d` and `-H`

**Review by ashahba — CHANGES_REQUESTED** [review]
Thanks @mkbhanda for this PR.
Most comments are minor.

**Review by ashahba — APPROVED** [review]
LGTM!

**Review by kevinintel — APPROVED** [review]

**Pull request merged**